doc/book/admin/instance_config.rst

```diff
@@ -84,11 +82,83 @@ entire application business logic in it. We, however, do not recommend this,
 since it clutters the instance file and leads to unnecessary copy-paste when
 you need to run multiple instances of an application.
 
+.. _admin-tt-preload:
+
+Preloading Lua scripts and modules
+----------------------------------
+
+Tarantool supports loading and running chunks of Lua code before the loading instance file.
+To load or run Lua code immediately upon Tarantool startup, specify the ``TT_PRELOAD``
+environment variable. Its value can be either a path to a Lua script, or a Lua module name:
+
+*   To run the Lua script ``script.lua`` from the ``preload/path/`` directory inside
+    the working directory in Tarantool before executing ``main.lua``:
```

Comment:
Maybe it's a matter of taste but I'd make this sentence full: `To run ..., set TT_PRELOAD as follows:`. I couldn't read it from the first attempt and looked for continuation :)